### PR TITLE
Doc: add an article on "how to access Apache APISIX Prometheus Metrics on k8s"

### PR DESCRIPTION
**Pull request by Chever-John**
As #953, this PR aims to add an article on "how to access Apache APISIX Prometheus Metrics on k8s"

### Type of change:

- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+```sh
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9180/apisix/admin/routes -H 'X-API-Key: edd1c9f034335f136f87ad84b625c8f1'
+
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9080/headers -H 'Host: test.prometheus.org'
+```
+
+## Conclusion
+
+This article describes the steps on how to use the `public-api` plugin to protect **Prometheus** and monitor the APISIX Ingress Controller. Currently, only some basic configurations are included, if you want to see some of the metrics displayed with **Grafana**, please refer to this [link](https://apisix.apache.org/zh/blog/2021/12/13/monitor-apisix-ingress-controller-with-prometheus/#). We will continue to polish and upgrade, add more metrics and integrate data surface APISIX metrics to bring you a better monitoring experience.
```
Comment (tao12345666333) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:111]: no, this is for monitoring APISIX not APISIX Ingress controller

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 1 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:111]: Got it!

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+Observability (monitoring functionality) has always played an important role in system maintenance. A good monitoring system can help engineers quickly understand the status of services running in production environments and can locate problems or give early warning of anomalies when they occur.
+
+Prometheus is a leading open-source project focused on metrics and alerting that has changed the way the world does monitoring and observability. And Apache APISIX has enhanced its support for Prometheus Metrics in recent releases, adding a new feature for use in conjunction with the `public-api` plugin. This article will introduce how to configure `public-api` to protect Prometheus to collect Apache APISIX 's Metrics data.
```
Comment (tokers) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:26]: Add a Prometheus website link.

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+## Initial Knowledge about `public-api`
+
+When users develop custom plugins in Apache APISIX, they can define some APIs (hereinafter referred to as public API) for the plugins. In practical application scenarios, the provided interface is for internal calls, rather than being open on the public network for anyone to call.
+
+Therefore, Apache APISIX has designed a `public-api` plugin. With this plugin, you can solve the pain points of using the public API. You can set a custom URI for the public API and configure any type of plugin. For more information about `public-api`, see the [public-api](https://apisix.apache.org/docs/apisix/plugins/public-api/) plugin's document.
+
+The main role of the `public-api` plugin in this document is to protect the URI exposed by Prometheus.
+
+**Note**: One thing we should note is that this feature is only available in APISIX version 2.13 and later.
+
```
Comment (tokers) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:37]: I think we don't have to introduce `public-api` here as it's APISIX's feature, we just need to add a link. Duplicated documentation will cause inconsistence.

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+## Begin to access Apache APISIX Prometheus Metrics
+
+### Step1: Install APISIX Ingress Controller
```
Comment (tokers) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:40]:
```suggestion
### Step 1: Install APISIX Ingress Controller
```

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+helm repo update
+kubectl create namespace ingress-apisix
+helm install apisix apisix/apisix --namespace ingress-apisix \
+ --set ingress-controller.enabled=true
+```
+
+After installation, please wait until all services are up and running. Specific status confirmation can be checked with the following command.
+
+```sh
+kubectl get all -n ingress-apisix
```
Comment (tokers) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:55]: Do we have any dedicated docs to introduce the way we install Apache APSIX Ingress Controller? @tao12345666333 

I don't think show the installation in a lot of docs is a good idea.

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+  monitoring:
+    prometheus:
+      enable: true
+```
+
+### Step 3: Enable `public-api` Plugin
+
+This is a basic routing setup, please note that further configuration should be done based on your local backend service information. The main solution concept is to use the `public-api` plugin to protect the routes exposed by Prometheus. For a more detailed configuration, you can refer to the [example](https://apisix.apache.org/zh/docs/apisix/plugins/public-api/#example) section of the `public-api` plugin.
+
+```yaml
```
Comment (tokers) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:77]: @tao12345666333 I think we can add an option to help users expose the Prometheus endpoint in the helm chart.

**Review by tokers — COMMENTED** [review]
@Chever-John Thanks for the document, Grammarly is a nice tool to help you write a better doc in English, would you like to use it fix some English syntax errors and make the paragraphs more formal.

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 5 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:55]: Yeah, we have!
"The hard way": https://apisix.apache.org/docs/ingress-controller/practices/the-hard-way

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 4 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:40]: Got it!

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 3 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:37]: Sry, I introduced `public-api` too much!

However, I think there is still some need to introduce `public-api`, indeed, this plugin affects the application of prometheus in APISIX.

**Comment by Chever-John** [comment]
also cc @navendu-pottekkat

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+Observability (monitoring functionality) has always played an important role in system maintenance. A good monitoring system can help engineers quickly understand the status of services running in production environments and can locate problems or give early warning of anomalies when they occur.
+
+Prometheus is a leading open-source project focused on metrics and alerting that has changed the way the world does monitoring and observability. And Apache APISIX has enhanced its support for Prometheus Metrics in recent releases, adding a new feature for use in conjunction with the `public-api` plugin. This article will introduce how to configure `public-api` to protect Prometheus to collect Apache APISIX 's Metrics data. You can learn more about Prometheus by clicking on this [link](https://prometheus.io/).
```
Comment (navendu-pottekkat) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:26]: Descriptive link text is always better: https://developers.google.com/style/link-text

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+
+## Initial Knowledge about `public-api`
+
+When users develop custom plugins in Apache APISIX, they can define some APIs (hereinafter referred to as public API) for the plugins. In practical application scenarios, the provided interface is for internal calls, rather than being open on the public network for anyone to call.
+
+Therefore, Apache APISIX has designed a `public-api` plugin. With this plugin, you can solve the pain points of using the public API. You can set a custom URI for the public API and configure any type of plugin. For more information about `public-api`, see the [public-api](https://apisix.apache.org/docs/apisix/plugins/public-api/) plugin's document.
```
Comment (navendu-pottekkat) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:32]: All APISIX objects should be capitalized. In this case it should be Plugin.

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+
+The main role of the `public-api` plugin in this document is to protect the URI exposed by Prometheus.
+
+**Note**: One thing we should note is that this feature is only available in APISIX version 2.13 and later.
+
+## Begin to access Apache APISIX Prometheus Metrics
+
+### Step 1: Install APISIX Ingress Controller
+
+First, we deploy Apache APISIX, ETCD, and APISIX Ingress Controller to a local Kubernetes cluster via Helm.
```
Comment (navendu-pottekkat) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:42]:
```suggestion
First, we deploy Apache APISIX, etcd, and APISIX Ingress Controller to a local Kubernetes cluster via Helm.
```

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+ --set ingress-controller.enabled=true
+```
+
+After installation, please wait until all services are up and running. Specific status confirmation can be checked with the following command.
+
+```sh
+kubectl get all -n ingress-apisix
+```
+
+If you have problems with the installation, you can click on this [link](https://apisix.apache.org/docs/ingress-controller/practices/the-hard-way) to get detailed instructions.
```
Comment (navendu-pottekkat) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:58]: Descriptive link text. See same comment above.

**Review by navendu-pottekkat — COMMENTED** [review]
I have left some comments. Overall, a good article 💯 

Feel free to use the Docusaurus [Admonitions](https://docusaurus.io/docs/next/markdown-features/admonitions) and other features as needed. They do make the reading experience better.

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 7 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:26]: Thank you so much for this great advice!

Comment (navendu-pottekkat) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md]: Thank Google's team for putting this together. Every guideline in it is solid. I have adopted most, if not all of it for our documentation guideline. There is a technical writing tutorial there which is very good.

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 8 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md]: Got it~!

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 9 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:42]: Got it~

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 8 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md]: Got it~

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 10 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md]: Got it~

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 5 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md]: I have bought the Grammarly package and have taken a lot of its advice.
Thanks for your great suggestions.

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 11]
```diff
@@ -0,0 +1,115 @@
+---
+title: How to access Apache APISIX Prometheus metrics on k8s
```
Comment (tao12345666333) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:2]:
```suggestion
title: How to access Apache APISIX Prometheus metrics on Kubernetes
```

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+
+```sh
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9180/apisix/admin/routes -H 'X-API-Key: edd1c9f034335f136f87ad84b625c8f1'
+
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9080/headers -H 'Host: test.prometheus.org'
+```
+
+## Conclusion
+
+This article describes how to use the `public-api` plugin to protect Prometheus and monitor the Apache APISIX. Currently, only some basic configurations include. If you want to see some of the metrics displayed with Grafana, please refer to this [link](https://apisix.apache.org/zh/blog/2021/12/13/monitor-apisix-ingress-controller-with-prometheus/#). We will continue to polish and upgrade, add more metrics and integrate data surface APISIX metrics to improve your monitoring experience.
```
Comment (tao12345666333) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:113]: This feature has nothing to do with APISIX Ingress, please remove this link

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 11 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:2]: Solve it~

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+Observability (monitoring functionality) has always played an essential role in system maintenance. A sound monitoring system can help engineers quickly understand the status of services running in production environments and locate problems or give early warning of anomalies when they occur.
+
+Prometheus is a leading open-source project focused on metrics and alerting that has changed the way the world does monitoring and observability. And Apache APISIX has enhanced its support for Prometheus Metrics in recent releases, adding a new feature for use in conjunction with the `public-api` plugin. This article will introduce how to configure `public-api` to protect Prometheus to collect Apache APISIX 's Metrics data. For more information, see Prometheus's official [website](https://prometheus.io/).
```
[the remark this answers is not in the document]
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:26]: Solved it~

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+
+Therefore, Apache APISIX has designed a `public-api` plugin. With this plugin, you can solve the public API's pain points. You can set a custom URI for the public API and configure any plugin. For more information, see the `public-api` Plugin's official [document](https://apisix.apache.org/docs/apisix/plugins/public-api/).
+
+The primary role of the `public-api` plugin in this document is to protect the URI exposed by Prometheus.
+
+**Note**: We should note that this feature is only available in APISIX version 2.13 and later.
+
+## Begin to access Apache APISIX Prometheus Metrics
+
+### Step 1: Install APISIX Ingress Controller
```
[the remark this answers is not in the document]
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:40]: Solve it~

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 12 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:113]: Sure, sir~

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+```
+
+### Step 3: Collect the Metrics
+
+Now you can then get the indicator parameters by requesting command access.
+
+```sh
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9180/apisix/admin/routes -H 'X-API-Key: edd1c9f034335f136f87ad84b625c8f1'
+
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9080/headers -H 'Host: test.prometheus.org'
```
Comment (tao12345666333) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:90]: You can show the output contents. 

This way we know what the expected result should be

**Review by tao12345666333 — COMMENTED** [review]
Left some comments

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+  name: default
+spec:
+  monitoring:
+    prometheus:
+      enable: true
+```
+
+### Step 2: Enable `public-api` Plugin
+
+Let's make a basic routing setup, and please note that further configuration should be done based on your local backend service information. The primary solution concept is to use the `public-api` plugin to protect the routes exposed by *Prometheus*. For a more detailed configuration, you can refer to the [example](https://apisix.apache.org/zh/docs/apisix/plugins/public-api/#example) section of the `public-api` plugin.
```
[the remark this answers is not in the document]
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:59]: OK~

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+  http:
+  - name: public-api
+    match:
+      hosts:
+      - test.prometheus.org
+      paths:
+      - /apisix/prometheus/metrics
+    backends:
+    ## Please notice that there must be your actual "serviceName" and "servicePort"
+    - serviceName: apisix-test-prometheus
```
[the remark this answers is not in the document]
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:76]: OK~

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+Observability (monitoring functionality) has always played an essential role in system maintenance. A sound monitoring system can help engineers quickly understand the status of services running in production environments and locate problems or give early warning of anomalies when they occur.
+
+*Prometheus* is a leading open-source project focused on metrics and alerting that has changed the way the world does monitoring and observability. APISIX has been used with the `public-api` plugin to enhance security in recent versions. For more information, see *Prometheus*'s official [website](https://prometheus.io/).
```
[the remark this answers is not in the document]
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:26]: OK~

**Comment by juzhiyuan** [comment]
Hi, @navendu-pottekkat, please have a review once more when you have time :)

**Comment by tao12345666333** [comment]
@Chever-John any update?

**Comment by Chever-John** [comment]
@tao12345666333 Yeah, it will be done today!

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 19]
```diff
@@ -0,0 +1,89 @@
...
+
+Now you can then get the indicator parameters by requesting command access.
+
+```sh
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9180/apisix/admin/routes -H 'X-API-Key: edd1c9f034335f136f87ad84b625c8f1'
+
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9080/headers -H 'Host: test.prometheus.org'
+```
+
+Then you will get the metrics you want.
```
Comment (tao12345666333) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:83]: ?

I don't think this sentence has any real meaning. 

As I pointed out in a previous comment, you need to post the output

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 19 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:83]: sure, actually, i can not reproduce the result at that time.
let me try again~

Comment (tao12345666333) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:83]: Well, if a document cannot be effectively guided, or cannot be reproduced according to the document, then the document will be meaningless

Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:83]: I have given the correct command to the expected result.
Please review it again, sir~
Thanks again~

**Comment by tao12345666333** [comment]
a kind reminder  @juzhiyuan @navendu-pottekkat  PTAL, thanks!  the content is correct

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 20]
```diff
@@ -0,0 +1,170 @@
+---
+title: How to access Apache APISIX Prometheus metrics on Kubernetes
```
Comment (juzhiyuan) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:2]: Would better have a description here, could do this in another PR.

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md [hunk 21]
```diff
@@ -0,0 +1,170 @@
...
+```yaml
+apiVersion: apisix.apache.org/v2beta3
+kind: ApisixClusterConfig
+metadata:
+  name: default
+spec:
+  monitoring:
+    prometheus:
+      enable: true
+```
```
Comment (juzhiyuan) [inline docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:45]: After enabling plugin, how to apply it to take effect?

**Review by juzhiyuan — APPROVED** [review]

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 21 above)
Comment (tao12345666333) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:45]: ping @Chever-John  PTAL

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 20 above)
Comment (tao12345666333) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:2]: ping @Chever-John  PTAL

**Review by navendu-pottekkat — APPROVED** [review]

File: docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md (hunk 21 above)
Comment (Chever-John) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:45]: > After enabling plugin, how to apply it to take effect?

Just add a command like this `kubectl apply -f xxx.yaml`?
or you mean the "### Step 3: Collect the Metrics"?

Comment (juzhiyuan) [reply docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md:45]: No worry, we could do this later. :) (Because other docs need this to improve SEO & DX as well
